packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -28,30 +27,17 @@ const useBigNumberConfig = (
         ? getFieldLabel(field)
         : fieldId && friendlyName(fieldId);
 
-    const [bigNumberLabel, setBigNumberName] = useState<
+    const [bigNumberLabel, setBigNumberLabel] = useState<
         BigNumber['label'] | undefined
-    >(bigNumberConfigData?.label || label);
+    >(bigNumberConfigData?.label);
 
-    const [bigNumberStyle, setStateBigNumberStyle] = useState<
+    const [bigNumberStyle, setBigNumberStyle] = useState<
         BigNumber['style'] | undefined
-    >(bigNumberConfigData?.style || undefined);
-    useEffect(() => {
-        setBigNumberName(bigNumberConfigData?.label || label);
-        setStateBigNumberStyle(bigNumberConfigData?.style || undefined);
-    }, [
-        resultsData,
-        bigNumberConfigData?.label,
-        label,
-        bigNumberConfigData?.style,
-    ]);
-
-    const setBigNumberLabel = useCallback((name: string | undefined) => {
-        setBigNumberName((prev) => name || prev);
-    }, []);
+    >(bigNumberConfigData?.style);
 
-    const setBigNumberStyle = useCallback((style: NumberStyle | undefined) => {
-        setStateBigNumberStyle(style);
-    }, []);
+    useEffect(() => {
+        setBigNumberLabel(bigNumberConfigData?.label);
```

Comment:
By removing the `setStateBigNumberStyle` I think is causing this bug: https://www.loom.com/share/92f7f31f27a4435cbb145c453478f6cc

Reply:
Interesting, I didn't test that particular scneario. I removed it because it was causing some other problems , specially when there was not label specified . I'll put it back so you can test it

Reply:
I reverted the style, you can test it now , I can see 2 problems

- On runQuery, style is resetted 
- When label is removed, style is resetted 

 ![Peek 2022-05-11 08-45](https://user-images.githubusercontent.com/1983672/167785887-26a22c24-d39a-4423-bf2d-664f561b3ae3.gif)

Reply:
and I don't think this is fixing your issue with the other tab... :thinking:  any suggestion ?

Reply:
Weird, I will test this locally to understand what is happening.

Reply:
Pushed a fix. The hook dependencies were wrong and causing unwanted calls.